dask/array/tests/test_fft.py

```diff
@@ -137,6 +137,7 @@ def test_wrap_ffts(modname, funcname, dtype):
 
     wfunc = fft_wrap(func)
     assert wfunc(darrc).dtype == func(nparrc).dtype
+    assert wfunc(darrc).shape == func(nparrc).shape
```

Comment:
So I looked at the comparison code and it is checking the shapes, but only after computing the dask arrays. The problem is the shape of the dask is wrong, but the shape of the computed result is correct. Can add a PR to compare these too. Though testing that case may prove tricky.